decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -73,6 +80,7 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
                                      y: self.view.frame.origin.y,
                                      width: self.view.frame.width,
                                      height: window.origin.y + window.height)
+             self.tableView.contentInset = UIEdgeInsets(top: 0,left: 0,bottom: 0,right: 0)
```

Comment:
Add an empty line space and the following comment before this line:
`// remove space at the bottom of table that was added when keyboard was displayed`